react-number-format/build.boot

```diff
@@ -0,0 +1,38 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2"  :scope "test"]
+                  [cljsjs/react "15.3.1-0"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "2.0.0")
```

Comment:
Use "2.0.0-alpha2" as the lib version and append just -0 as the build identifier to version.